src/Module.Server/ModuleData/dtv/dtv_characters.xml

```diff
@@ -3327,6 +3327,424 @@
+      <skill id="Polearm" value="0" />
+      <skill id="Bow" value="0" />
+      <skill id="Crossbow" value="0" />
+      <skill id="Throwing" value="0" />
+    </skills>
+    <Equipments>
+      <EquipmentRoster>
+        <equipment slot="Head" id="Item.crpg_rus_helm_regular_v3_h0" />
+        <equipment slot="Cape" id="Item.crpg_bearskin_v2_h0" />
+        <equipment slot="Body" id="Item.crpg_battania_woodland_chainmail_v3_h0" />
+        <equipment slot="Gloves" id="Item.crpg_plated_strip_gauntlets_v2_h0" />
+        <equipment slot="Leg" id="Item.crpg_plated_strip_boots_v2_h0" />
+        <equipment slot="Item1" id="Item.crpg_decorated_elite_scimitar_v1_h0" />
+        <equipment slot="Item2" id="Item.crpg_heavy_heater_shield_v3_h0" />
+      </EquipmentRoster>
+	  <EquipmentRoster>
+	    <equipment slot="Head" id="Item.crpg_rus_helm_regular_v3_h0" />
+        <equipment slot="Cape" id="Item.crpg_bearskin_v2_h0" />
+        <equipment slot="Body" id="Item.crpg_vlandia_chainmail_v2_h0" />
+        <equipment slot="Gloves" id="Item.crpg_plated_strip_gauntlets_v2_h0" />
+        <equipment slot="Leg" id="Item.crpg_plated_strip_boots_v2_h0" />
+        <equipment slot="Item0" id="Item.crpg_steel_pick_v1_h0" />
+        <equipment slot="Item2" id="Item.crpg_heavy_heater_shield_v3_h0" />
+    </EquipmentRoster>
+    </Equipments>
+  </NPCCharacter>
+  <NPCCharacter id="crpg_dtv_greyorder_kuyak_two_handed"
+            name="{=}Grey Order Kuyak Hero" race="light_bot">
+    <face>
+      <BodyProperties version="4"
+                      age="30"
+                      weight="0.1"
+                      build="0.1"
+                      key="0020C80FDA00061B0776678E53648686588CA83378A87F2099667D788BC3324503777635076699853444400000000000000000003EFC3002000000004EDC3002" />
+    </face>
+    <skills>
+      <skill id="Strength" value="15" />
+      <skill id="Agility" value="33" />
+      <skill id="IronFlesh" value="5" />
+      <skill id="PowerStrike" value="5" />
+      <skill id="PowerDraw" value="0" />
+      <skill id="Athletics" value="232" />
+      <skill id="PowerThrow" value="0" />
+      <skill id="Riding" value="0" />
+      <skill id="WeaponMaster" value="11" />
+      <skill id="MountedArchery" value="0" />
+      <skill id="Shield" value="0" />
+      <skill id="OneHanded" value="0" />
+      <skill id="TwoHanded" value="220" />
+      <skill id="Polearm" value="0" />
+      <skill id="Bow" value="0" />
+      <skill id="Crossbow" value="0" />
+      <skill id="Throwing" value="0" />
+    </skills>
+    <Equipments>
+      <EquipmentRoster>
+        <equipment slot="Head" id="Item.crpg_vaegir_helm_v2_h0" />
+        <equipment slot="Cape" id="Item.crpg_wolf_shoulder_v2_h0" />
+        <equipment slot="Body" id="Item.crpg_battania_woodland_chainmail_v3_h0" />
+        <equipment slot="Gloves" id="Item.crpg_plated_strip_gauntlets_v2_h0" />
+        <equipment slot="Leg" id="Item.crpg_plated_strip_boots_v2_h0"" />
```

Comment:
There is an extra " on line:
3413
3546
3585
3624
3664
3703
Almost all the lines with boots

Reply:
You missed lin 3546 :)